app/routes/_gcn.docs.contributing.project/route.mdx

```diff
@@ -0,0 +1,196 @@
+---
+handle:
+  breadcrumb: Project Board
+---
+
+import { Table } from '@trussworks/react-uswds'
+
+import issueScreenshot from './issue.png'
+import openTicketScreenshot from './openTicket.png'
+import projectBoardScreenshot from './projectBoard.png'
+import ticketMetadata from './ticketMetadata.png'
+
+# Github Project Board
+
+[GCN Project Board](https://github.com/orgs/nasa-gcn/projects/6)
+
+## Philosophy
+
+We use github Projects to organize our development process.
```

Comment:
```suggestion
We use GitHub Projects to organize our development process.
```

Reply:
Please capitalize as GitHub, not github, here and throughout this patch.